ts/store/reducers/entities/readTransactions.ts

```diff
@@ -54,3 +55,16 @@ export const getUnreadTransactionsSelector = createSelector(
       txs.filter(_ => _ !== undefined && transactionsRead[_.id] === undefined)
     )
 );
+
+/**
+ * This selector returns the number of unread transaction
+ * only if the current user has the email validated.
+ */
+export const getSafeUnreadTransactionsNumSelector = (state: GlobalState) => {
+  const transactions = getUnreadTransactionsSelector(state);
+  const isEmailValidated = isProfileEmailValidatedSelector(state);
```

Comment:
❓**question**: I know it was already like this, but do we know why the email has to be validated to compute the transactions' number?

Reply:
I thought it was something related to privacy, or at least I took it for granted. I thought that maybe that information wasn't available inside the app without a validated email, and so the `accessibilityLabel` would need to respect that too. Looking at the code a second time tho, this may be not the case (?)